charts/actions-runner-controller/values.yaml

```diff
@@ -8,11 +8,10 @@ replicaCount: 1
 
 syncPeriod: 10m
 
-controller:
-  enableLeaderElection: true
-  # Specifies the controller id for leader election.
-  # Must be unique if more than one controller installed.
-  #leaderElectionId: "actions-runner-controller"
+enableLeaderElection: true
```

Comment:
@rofafor Hey! Thanks again for #806. I'm moving this to the top-level for consistency with existing keys like `syncPeriod`.